Resources/Prototypes/_LostParadise/Entities/Objects/Specific/Robotic/borg_modules_syndi_assault.yml

```
@@ -0,0 +1,47 @@
+  parent: [ BaseBorgModuleSyndicateAssault, BaseProviderBorgModule ]
+  name: assault cyborg module
+  description: There is no wall that can stop you.
+  components:
+    - type: Sprite
+      sprite: _LostParadise/Objects/Specific/Robotics/borgmodule.rsi
+      layers:
+      - state: syndicate
+      - state: icon-assault-ram
+    - type: ItemBorgModule
+      items:
+      - LPPAssaultRam
```

yamllint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)